### PR TITLE
Bibash sitoula

### PR DESCRIPTION
**Pull request by Bibash-dev**

### CODE REVIEW

File: Basic/bibash.html [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+                    <div class="row">
+                        <a href="" class="word"><img src="image/Dot-Icon-Orange.webp" width="20px">Digital Marketing</a>
+                        <p class="animation" data-aos="fade-up">Digital Marketing is the initiative of taking your marketing efforts to a digital medium where it can reach each and every one of your potential customers.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
```
Comment (chandan-g-bhagat) [inline Basic/bibash.html:105]: remove this unwanted line

File: Basic/bibash.html [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+                    <a href="" class="blog1" id="list"><img src="image/index.png" width="20px"> Do You Really Need a Website?
+                    </a><br><br>
+                    <a href="" class="blog1" id="list"><img src="image/index.png" width="20px"> March Performer of the Month: Pranish Maharjan
+                    </a><br><br>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
```
Comment (chandan-g-bhagat) [inline Basic/bibash.html:209]: remove this line

**Review by chandan-g-bhagat — CHANGES_REQUESTED** [review]

File: Basic/bibash.html (hunk 1 above)
Comment (Bibash-dev) [reply Basic/bibash.html:105]: resolved

**Review by Nabinkc10 — COMMENTED** [review]
resolved

**Review by chandan-g-bhagat — APPROVED** [review]
approved

**Review by asbinaryal — APPROVED** [review]
approved

File: Basic/bibash.html (hunk 2 above)
Comment (Rijwan12) [reply Basic/bibash.html]: resolved

**Pull request merged**